detect_secrets/core/usage.py

```diff
@@ -78,10 +127,36 @@ def add_console_use_arguments(self):
         return self
 
     def parse_args(self, argv):
-        output = self.parser.parse_args(argv)
-        PluginOptions.consolidate_args(output)
+        # We temporarily remove '--help' so that we can give the full
```

Comment:
I'm not sure if I understand this, can you explain?

Reply:
If you do `--help` with `--custom-plugins`, you want to see the options for e.g. `--no-custom-detector`, if we just parse `--help` before `--custom-plugins`, it will SystemExit and give the help message without any options from the custom plugins.